ehharvey/conglobo
# K8s in container

* Update devcontainer.json
* Adds k8s to the devcontainer

# Test instructions
- [ ] Clone branch
- [ ] Rebuild container
- [ ] Run build task (ctrl shift p)
    - You will want to run the default build task (in the VS Code UI it'll show up as just `Run build task`
    - Otherwise, you can select `Run task` and run the `Deploy Conglobo Management App` task
    - The build task will take a while
    - The build task will spawn an extra terminal called `Forward Kubernetes Ingress to port 8080`. This terminal runs a daemon that lets you access the K8s cluster. Once the build completes, you can visit the conglobo app on port `8080` (make sure it is being forwarded)
- [ ] The API (lines 31-47) will create a test app. Check the route `testapp`. It should resolve to a test app

<img width="438" alt="image" src="https://user-images.githubusercontent.com/36806437/228022145-3f9dd513-bed8-4eff-a3df-8e7370b7d1c8.png">
